modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -4,14 +4,19 @@
 // migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc
 // migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc
 [id='migration-adding-replication-repository-to-cam_{context}']
-= Adding a Repository to the CAM web console
+= Adding a replication repository to the CAM web console
 
-You can add an S3 repository to the CAM web console.
+You can add a replication repository to the CAM web console.
+
+.Prerequisites
+
+* The storage provider must support the S3 API.
```

Comment:
Since we expanded the types of Object Store, this is no longer true.

The replication repository can refer to:
 - AWS S3 (or a generic S3, like NooBaa, minio, ceph s3, etc)
 - Azure Blob 
 - Google Cloud Storage

Reply:
I am deleting this line (about S3 API) and moving comment to "Configuring a replication repository"